### PR TITLE
Bug report and feature request templates updating

### PR DESCRIPTION
**Pull request by Pavel-Lapatin**
#### CHANGE TYPE

 - [ ] Major
 - [ ] Minor
 - [x] Patch

#### GITHUB ISSUE LINK

#26 
#### IMPACT, DEPENDENCIES AND TEST CONSIDERATIONS

Bug report and feature request were updated: 
  - 1. Prioraty was removed from bug report.
  - 2. Acceptance creteria was removed from feature request.

Testing is not required for these fixes.

### CODE REVIEW

File: src/GodelTech.Microservices.Core/GodelTech.Microservices.Core.csproj [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <Authors>Andrei Salanoi</Authors>
     <RepositoryUrl>https://github.com/GodelTech/GodelTech.Microservices.Core</RepositoryUrl>
     <PackageIconUrl>https://www.gravatar.com/avatar/839234621070de51e7b9cabd5ceee8fe?s=64</PackageIconUrl>
-    <VersionPrefix>1.3.0</VersionPrefix>
+    <VersionPrefix>1.3.1</VersionPrefix>
```
Comment (DiamondDragon) [inline src/GodelTech.Microservices.Core/GodelTech.Microservices.Core.csproj]: @Pavel-Lapatin i think we can revert this change because we are not going to re-build package. Our changes are not related to package. These are infrastructure changes.

Comment (Pavel-Lapatin) [reply src/GodelTech.Microservices.Core/GodelTech.Microservices.Core.csproj]: Change was reverted

**Review by DiamondDragon — APPROVED** [review]
